unpick-definitions/entity_model_parts.unpick

```diff
@@ -0,0 +1,74 @@
+constant entity_model_parts net/minecraft/client/render/entity/model/EntityModelPartNames NOSE
+constant entity_model_parts net/minecraft/client/render/entity/model/EntityModelPartNames ARMS
+constant entity_model_parts net/minecraft/client/render/entity/model/EntityModelPartNames TAIL
+constant entity_model_parts net/minecraft/client/render/entity/model/EntityModelPartNames CUBE
+constant entity_model_parts net/minecraft/client/render/entity/model/EntityModelPartNames BEAK
+constant entity_model_parts net/minecraft/client/render/entity/model/EntityModelPartNames BACK_FIN
+constant entity_model_parts net/minecraft/client/render/entity/model/EntityModelPartNames MANE
+constant entity_model_parts net/minecraft/client/render/entity/model/EntityModelPartNames NECK
+constant entity_model_parts net/minecraft/client/render/entity/model/EntityModelPartNames MOUTH
+constant entity_model_parts net/minecraft/client/render/entity/model/EntityModelPartNames HEAD
+constant entity_model_parts net/minecraft/client/render/entity/model/EntityModelPartNames HAT
+constant entity_model_parts net/minecraft/client/render/entity/model/EntityModelPartNames BODY
+constant entity_model_parts net/minecraft/client/render/entity/model/EntityModelPartNames HAT_RIM
+constant entity_model_parts net/minecraft/client/render/entity/model/EntityModelPartNames JACKET
+constant entity_model_parts net/minecraft/client/render/entity/model/EntityModelPartNames TOP_GILLS
+constant entity_model_parts net/minecraft/client/render/entity/model/EntityModelPartNames LEFT_GILLS
+constant entity_model_parts net/minecraft/client/render/entity/model/EntityModelPartNames RIGHT_GILLS
+
+target_method net/minecraft/client/model/ModelPart getChild (Ljava/lang/String;)Lnet/minecraft/client/model/ModelPart;
+	param 1 entity_model_parts
+target_method net/minecraft/client/model/ModelPartBuilder cuboid (Ljava/lang/String;FFFIIILnet/minecraft/client/model/Dilation;II)Lnet/minecraft/client/model/ModelPartBuilder;
+	param 1 entity_model_parts
+target_method net/minecraft/client/model/ModelPartBuilder cuboid (Ljava/lang/String;FFFIIIII)Lnet/minecraft/client/model/ModelPartBuilder;
+	param 1 entity_model_parts
+target_method net/minecraft/client/model/ModelPartBuilder cuboid (Ljava/lang/String;FFFFFF)Lnet/minecraft/client/model/ModelPartBuilder;
+	param 1 entity_model_parts
+target_method net/minecraft/client/model/ModelPartBuilder cuboid (Ljava/lang/String;FFFFFFLnet/minecraft/client/model/Dilation;)Lnet/minecraft/client/model/ModelPartBuilder;
+	param 1 entity_model_parts
+target_method net/minecraft/client/model/ModelPartData getChild (Ljava/lang/String;)Lnet/minecraft/client/model/ModelPartData;
+	param 1 entity_model_parts
+target_method net/minecraft/client/model/ModelPartData addChild (Ljava/lang/String;Lnet/minecraft/client/model/ModelPartBuilder;Lnet/minecraft/client/model/ModelTransform;)Lnet/minecraft/client/model/ModelPartData;
+	param 1 entity_model_parts
```

Comment:
Wait, I think these are supposed to be `param 0` in unpick format, even though these are instance methods

Reply:
(At least that's how it worked when I last tested in #2246)

Reply:
fml I tested everything but that. It's confusing to have different param indexing

Reply:
Indeed! Unpick is quite confusing. Even its wiki incorrectly described its format: when you need to specify constant value and desc, say value `255` and type `I`, you should have `255 I` than `I 255` as the wiki suggests. So yeah, this is indeed painstaking!

Reply:
> Wait, I think these are supposed to be `param 0` in unpick format, even though these are instance methods

They're parameter indices, not local variable indices. So there's no difference between static and instance methods.

>  Even its wiki incorrectly described its format: when you need to specify constant value and desc, say value 255 and type I, you should have 255 I than I 255 as the wiki suggests.

If you find an issue with the docs, please report it. Though you can't do that right now, since I archived WeaveMC/unpick last night.